source/templates.tex

```diff
@@ -4386,8 +4386,8 @@
 
 \pnum
 A \grammarterm{qualified-id}
```

Comment:
*qualified-id* is just an expression defined in [expr.prim.id.qual], how would it be anything others other than an expression? As we discussed in https://github.com/cplusplus/draft/issues/4791#issuecomment-892908042.

Reply:
A qualified-id is also used in declarative contexts, e.g.

`void N::C::f() { ..}`

`N::C::f` is a _qualified-id_.